_translator-files/po/de/general.po

```diff
@@ -85,13 +86,13 @@ msgstr "Mehr..."
 #: ../wiki/en/misc/general.yml:0
 #, no-wrap
 msgid "– Jamulus Knowledge Base"
-msgstr "– Jamulus Knowledge Base"
+msgstr "– Jamulus Wissensdatenbank"
 
 #. type: Hash Value: nav altJamulusIcon
 #: ../wiki/en/misc/general.yml:0
 #, no-wrap
 msgid "Jamulus Icon. Links to homepage"
-msgstr "Jamulus Icon. Links to homepage"
+msgstr "Jamulus Icon. Link zur Hauptseite"
```

Comment:
Hauptseite --> Startseite (at least that's what I usually read)

Reply:
Corrected